### ISSUE TITLE
pulp container pull-through

### ISSUE THREAD

**Issue by katrinm86** [pulp/squeezer#178]
Hi there! 

I just wanted to make sure if I'm not getting something or if the rpm container pull-through configuration is really missing at the moment?

If yes: how would I go about to add it? I've seen that you also provide an "api_call" method that takes in the operation id of the api ... does it mean that I would need to do all the GET/POST/UPDATE calls myself? How hard would it be to add it using the PulpRemoteAnsibleModule?

**Comment by mdellweg** [comment]
Yes, squeezer is not getting sufficient attention to always be on top of all features. I am not really an expert for pull through, but i think it can be slightly different for each pulp plugin. Are you looking for pull through with the container registry, or rpm?
It may help to understand if you can describe how to set up your desired pull-through cache with pulp-cli.

**Comment by katrinm86** [comment]
Yeah, sorry, I totally forgot the context. So I'm trying to setup a couple of container pull-through endpoints. 

As far as I could see yesterday they did not yet implement that in the pulp-cli: https://github.com/pulp/pulp-cli/issues/1034 so I guess if one would want to add this the official way they would need to add it there first and then to squeezer :)

**Comment by mdellweg** [comment]
Yes, that sounds reasonable. The container pull-through distribution is a new type of object.
You can try to use the api_call module as a stop gap, if you feel adventurous and have a good grip at how the api works.
